### PR TITLE
Rancher: Add RKE2 multimachine test and remove RKE1

### PR DESCRIPTION
**Pull request by pdostal**
This multi machine test installs Kubernetes cluster of one master and two nodes via [rancher/rke2](https://github.com/rancher/rke2).
- Verification run: [JeOS-15-SP3](http://pdostal-server.suse.cz/tests/14224), [Tumbleweed](http://pdostal-server.suse.cz/tests/14227)

### CODE REVIEW

**Comment by pdostal** [comment]
cc: @jlausuch, @grisu48

File: tests/rancher/rancher_rke2_master.pm [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+use Utils::Systemd qw(systemctl disable_and_stop_service);
+use rancher::utils 'kubectl_basic_test';
+use mm_network 'setup_static_mm_network';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    # All nodes are online with SSH enabled
+    barrier_create('networking_prepared', 3);
```
Comment (grisu48) [inline tests/rancher/rancher_rke2_master.pm:23]: Is this barrier really needed?

File: tests/rancher/rancher_rke2_worker.pm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    assert_script_run('echo -en "server: https://10.0.2.99:9345\ntoken: " | tee /etc/rancher/rke2/config.yaml');
+
+    barrier_wait('cluster_prepared');
+    exec_and_insert_password("scp -o StrictHostKeyChecking=no  10.0.2.99:/var/lib/rancher/rke2/server/node-token ~/");
+    assert_script_run("cat ~/node-token >> /etc/rancher/rke2/config.yaml");
+    assert_script_run("cat /etc/rancher/rke2/config.yaml");
+    systemctl('enable --now rke2-agent.service', timeout => 900);
+
+    barrier_wait('cluster_deployed');
+    assert_script_run("ps aux");
```
Comment (grisu48) [inline tests/rancher/rancher_rke2_worker.pm:45]: Leftover?

**Review by grisu48 — COMMENTED** [review]
Two comments, but overall good work!

**Pull request closed without merging**